dev/NavigationView/NavigationView.cpp

```diff
@@ -4625,7 +4625,7 @@ void NavigationView::UpdateTitleBarPadding()
             {
                 if (IsOverlay())
                 {
-                    thickness = winrt::ThicknessHelper::FromLengths(c_backButtonWidth, 0, 0, 0);
+                    thickness = winrt::ThicknessHelper::FromLengths(c_backButtonHeight, 0, 0, 0);
```

Comment:
This change will **not** break the old styling, as `c_backButtonHeight` and `c_backButtonWidth` were the same.

Reply:
Why did you have to make this change, its weird to use the height as the width for the thickness.

Reply:
These are constant values which are both set to 40, so I suspect if you undo this change, it should not affect anything.

Reply:
Reverted